beacon-chain/state/state-native/getters_withdrawal.go

```diff
@@ -132,9 +134,19 @@ func (b *BeaconState) ExpectedWithdrawals() ([]*enginev1.Withdrawal, uint64, err
 				return nil, 0, fmt.Errorf("could not retrieve balance at index %d: %w", w.Index, err)
 			}
 			hasSufficientEffectiveBalance := v.EffectiveBalance() >= params.BeaconConfig().MinActivationBalance
-			hasExcessBalance := vBal > params.BeaconConfig().MinActivationBalance
+			var totalWithdrawn uint64
+			for _, wi := range withdrawals {
+				if wi.ValidatorIndex == w.Index {
+					totalWithdrawn += wi.Amount
+				}
+			}
+			balance, err := mathutil.Sub64(vBal, totalWithdrawn)
+			if err != nil {
+				return nil, 0, fmt.Errorf("failed to subtract balance %d with total withdrawn %d", vBal, totalWithdrawn)
```

Comment:
```suggestion
				return nil, 0, fmt.Errorf("failed to subtract balance %d with total withdrawn %d: %w", vBal, totalWithdrawn, err)
```

Reply:
let me use wrap instead to follow the same as below i blanked on this